server/api/endpoints/shop_endpoints/orders.py

```diff
@@ -93,7 +93,7 @@ def get_price_rules_total(order_items):
 def get_multi(
     response: Response,
     common: dict = Depends(common_parameters),
-    current_user: UserTable = Depends(cognito_eu.auth_required),
+    current_user: UserTable = Depends(auth_required),
```

Comment:
I see this happening in a few places in the code, but I don't think `current_user` is actually a value from the `UserTable` and is a `CognitoToken` instead.

So far it is fine because the value isn't used, but maybe we should update these variables/types?